ebmbot/job_configs.py

```diff
@@ -241,43 +239,44 @@
             },
         ],
     },
-    "teamrap": {
-        "python_file": "project_report.py",
+    "report": {
```

Comment:
If I understand correctly, I think these changes actually belong in the _next_ commit – not that it makes any real difference to anything.